<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 64, 64);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_id">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 1.7px solid black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_id">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>103</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(64, 64, 64);</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;

border: 1.7px solid rgb(45, 45, 185);
background-color: rgb(191, 191, 191);</string>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_pw">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>144</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(64, 64, 64);</string>
    </property>
    <property name="text">
     <string>PW</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_pw">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 1.7px solid black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_newuser">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;

border: 1.7px solid rgb(45, 45, 185);
background-color: rgb(191, 191, 191);</string>
    </property>
    <property name="text">
     <string>New User</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 30pt &quot;나눔명조 ExtraBold&quot;;

color: rgb(146, 208, 80);
background-color: rgb(64, 64, 64);</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>l_id</zorder>
   <zorder>btn_login</zorder>
   <zorder>l_pw</zorder>
   <zorder>txt_pw</zorder>
   <zorder>btn_newuser</zorder>
   <zorder>title</zorder>
   <zorder>txt_id</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
